ide/coqide/protocol/xmlprotocol.ml

```diff
@@ -12,7 +12,15 @@
 
 (** WARNING: TO BE UPDATED WHEN MODIFIED! *)
 
-let protocol_version = "20200911"
+(** Changelog
+
+    - 20210409: support for ltac debugger:
```

Comment:
Shouldn't that go somewhere else, e.g. in the `xml-protocol.md` file?

Reply:
I added info in `xml-protocol.md` and dropped this comment.